cmd/nerdctl/volume_list.go

```diff
@@ -87,7 +87,7 @@ func volumeLsAction(cmd *cobra.Command, args []string) error {
 func getVolumes(cmd *cobra.Command, globalOptions types.GlobalCommandOptions) (map[string]native.Volume, error) {
 	volumeSize, err := cmd.Flags().GetBool("size")
 	if err != nil {
-		return nil, err
+		volumeSize = false
```

Comment:
Could you add a comment line to explain this?

Reply:
Thanks @AkihiroSuda 
The comment line has been added.